src/transport/message_handler.rs

```diff
@@ -42,20 +45,48 @@ pub(crate) trait ProtocolMessageHandler {
 ///
 /// Allows end-user to configure options when using the [`LiquidityManager`]
 /// to provide liquidity services to clients.
-pub struct LiquidityProviderConfig;
+pub struct LiquidityProviderConfig {
+	/// Optional configuration for jit channels
+	/// should you want to support them
+	pub jit_channels: Option<JITChannelsConfig>,
+}
+
+/// Configuration options for jit channels
+/// A configuration used for the creation of Just In Time Channels.
+pub struct JITChannelsConfig {
```

Comment:
yeah, for LSP side might want to include `min_payment_size_msat` and `max_payment_size_msat`.  Though I wonder if we'll just need to provide overrides anyway.

for client side maybe we let them pick a 'strategy' for how they want to select a fee from the menu automatically and not surface opening fee params at all.  Allow them to set min/max fee ranges and/or target lifetime 🤷  Seems like it might get complicated but I don't imagine a lot of wallets (or LSPs for that matter) surfacing a fee menu to users.